scipy/optimize/__init__.py

```diff
@@ -86,6 +86,7 @@
 
    leastsq - Minimize the sum of squares of M equations in N unknowns
    nnls - Linear least-squares problem with non-negativity constraint
+   lsq_linear - Linear least-squares problem with arbitrary bound constraints
```

Comment:
I use "bound" and "bounds" everywhere in the code. "Bound constrained optimization" seems to be a valid and widely used term.